core/new-gui/src/app/dashboard/service/auth-guard/auth-guard.service.ts

```diff
@@ -0,0 +1,14 @@
+﻿import { Injectable } from "@angular/core";
+import { Router, CanActivate } from "@angular/router";
+import { UserService } from "../../../common/service/user/user.service";
+@Injectable()
+export class AuthGuardService implements CanActivate {
```

Comment:
Please add a class block comment to describe this service.  You can refer to other services' documentation.

Reply:
commented